### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

File: locale/nl/LC_MESSAGES/clock-override.po [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: Heimen Stoffels <vistausss@outlook.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: nl\n"
+
+#: prefs.js:51
+msgid "Text to display instead of the clock"
+msgstr "De in plaats van de klok te tonen tekst"
```
Comment (lode) [inline locale/nl/LC_MESSAGES/clock-override.po:23]: This one is a bit hard to parse I think. Maybe something like:

```suggestion
msgstr "De te tonen tekst in plaats van de klok"
```

Or:

```suggestion
msgstr "De tekst op de plek van de klok"
```

Comment (Vistaus) [reply locale/nl/LC_MESSAGES/clock-override.po:23]: I've never heard any user complaints about such sentences, but I agree that "De tekst op de plek van de klok" is better. Thanks for your collaboration! :)

**Review by lode — APPROVED** [review]
Nice!
